profiles/default/linux/amd64/13.0/desktop/proaudio/gnome/parent

```diff
@@ -0,0 +1 @@
+/usr/portage/profiles/default/linux/amd64/13.0/desktop/gnome
```

Comment:
Unfortunately, it just doesn't work:
```
# repoman
!!! Unable to parse profile: '/etc/make.profile'
!!! ParseError: Parent '/var/lib/layman/proaudio/profiles/default/linux/amd64/13.0/desktop/proaudio/gentoo:default/linux/amd64/13.0/desktop' not found: '/var/lib/layman/proaudio/profiles/default/linux/amd64/13.0/desktop/proaudio/parent'
```
I get the same error with portage. On bugzilla, it is at least 3 repoman bugs related to PORTDIR and PORTDIR_OVERLAY. The consequence is we must use the old syntax.